src/aleph/db/accessors/messages.py

```diff
@@ -87,7 +89,15 @@ def make_matching_messages_query(
         select_stmt = select_stmt.where(MessageDb.sender.in_(addresses))
     if chains:
         select_stmt = select_stmt.where(MessageDb.chain.in_(chains))
+    if message_types:
+        if len(message_types) == 1:
+            select_stmt = select_stmt.where(MessageDb.type == message_types[0])
+        else:
+            select_stmt = select_stmt.where(MessageDb.type.in_(message_types))
     if message_type:
+        warnings.warn(
+            "Warning: `msgType`/`message_type` query parameter is deprecated in favor of `msgTypes`\`message_types` and will be removed in future versions."
+        )
```

Comment:
This warning is useless as it will only appear in our logs. We need OpenAPI docs, some day.